### DISCUSSION TITLE
Help me

### DISCUSSION THREAD

**Discussion by Bayu246** [community/community#36751] [Other Feature Feedback, Questions, & Ideas]
Please help me, i can't login aptoslabs.com via github
![Screenshot_20221020_063747](https://user-images.githubusercontent.com/82433113/196892395-f0210259-66d1-4ff8-84e5-5538c413b7bf.jpg)

**Comment by Ghostbird** [comment]
It seems that this is a problem that Aptoslabs should fix on their side. What I think is that Github has reason to not trust Aptoslabs with your information, hence people can no longer login there with Github. If my interpretation is correct, the solution is that Aptoslabs should contact Github and ask them what they should do to restore trust.

**Reply by queenofcorgis** [reply]
Hi @Bayu246,

Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Aptoslabs. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
